tests/unit/test_darray.c

```diff
@@ -1,7 +1,5 @@
-/**
- * @file
+/*
```

Comment:
This is not really a working test. If you look, you will see it does nothing with darrays (yet). It is just the start of the darray test I am writing. (The same is true of test_darray_async.c) They started as code copied from a test creating a more complex test file. In this first darray outing I am going to keep it very simple. ;-)